### PR TITLE
Create a switch control for all checkable toolbar actions

### PR DESCRIPTION
**Pull request by bmonio**

### CODE REVIEW

File: src/controls/Switch.qml [hunk 1]
```diff
@@ -43,8 +52,8 @@ Controls.Switch {
             backgroundColor: control.enabled ? control.checked ? control.color
                                                                : darkBackground ? "#BDBDBD"
                                                                                 : "#FAFAFA"
-                                             : darkBackground ? "#424242"
```
Comment (Issam-Wakidi) [inline src/controls/Switch.qml:46]: can you please restore indentation as it was before.

File: src/controls/Switch.qml [hunk 2]
```diff
@@ -23,6 +23,15 @@ import Material 0.3
 Controls.Switch {
     id: control
 
+    property Action action
```
Comment (Issam-Wakidi) [inline src/controls/Switch.qml:29]: can you add a property that flags the display of checkable actions as switch. For example : property bool displayCheckableActionAsSwitch : false

Comment (Issam-Wakidi) [inline src/controls/Switch.qml:29]: and provide a one or two lines doc for this property

Comment (bmonio) [inline src/controls/Switch.qml:29]: I prefer to use the checkable property; Imagine I set checkable to false and displayCheckableActionAsSwitch to true, the switch behavior will be weird

**Pull request merged**